CivilizationMapParser/Program.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace CivilizationMapParser
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			ReadSourceFile(AskForPath());
+			WriteRegionTileInfo();
+		}
+
+		public static List<RegionTile> Regions = new List<RegionTile>();
+
+		public static string AskForPath()
+		{
+			string path;
+			Console.WriteLine("Provide path to Source file below:");
+			return path = Console.ReadLine();
+		}
+
+		public static void ReadSourceFile(string path)
+		{
+			string[] lines = File.ReadAllLines(path);
+			RegionTile Region = null;
```

Comment:
in the loop on the first iteration we initialize this `Region`. Why don't do it here and remove if-condition from the loop?